lib/dataset-tools/evaluation/sentence_evaluator.js

```diff
@@ -186,6 +187,7 @@ class SentenceEvaluator {
         const goldFunctions = untypedTargetCode.map((code) => Array.from(getFunctions(code)));
         const goldDevices = untypedTargetCode.map((code) => Array.from(getDevices(code)));
```

Comment:
Actually, are you going to fix this function so it doesn't return the dialogue policy name as a "device"?

Reply:
I was hesitant to do that because I wasn't sure if this function was being used elsewhere, possibly relying on this behavior. But I can do that. I'll also look into the tests - another PR or should it go together with this one?

Failed the linter. Fixing it now.

Reply:
Oh, I grep'd it and seems like it'll be fine to change this. Doing that now too